packages/connectivity/src/scp-cf/destination/destination-from-vcap.spec.ts

```diff
@@ -93,18 +182,56 @@ function mockServiceBindings() {
 }
 
 const serviceBindings = {
+  xsuaa: [xsuaaBindingMock],
   'business-logging': [
     {
       name: 'my-business-logging',
+      label: 'business-logging',
+      tags: [
+        'business-logging',
+        'logging',
+        'com.sap.appbasic.businesslogs',
+        'comsapappbasicbusinesslogs'
+      ],
       credentials: {
         writeUrl: 'https://business-logging.my.example.com',
         uaa: {
-          clientid: 'CLIENT_!_|_!_ID',
+          clientid: 'clientIdBusinessLogging',
```

Comment:
[q] Does this change remove testing this string with special characters? (assuming special characters are a sensible test case)

Reply:
no I just wanted to have different names for each client secret. I think with the string representation we do not have problem or test need for special characters.